src/Compilers/Core/Portable/Symbols/IArrayTypeSymbol.cs

```diff
@@ -47,6 +47,11 @@ public interface IArrayTypeSymbol : ITypeSymbol
         /// </summary>
         ITypeSymbol ElementType { get; }
 
+        /// <summary>
+        /// Gets the nullability of the elements stored in the array.
+        /// </summary>
+        Nullability ElementNullability { get; }
```

Comment:
So, i'm not disagreeing with this approach.  But how come this is not necessarily part of the ITypeSymobl itself at this projected level?  

For example, there is no "IsElementTypeDynamic" even though "Dynamic" and "Object" are effectively the same under the covers.  INstead, we represent each of those with different types.  Why not the same here?